### PR TITLE
enh(chore): Use dayjs instead of moment to save up bundle space and enhance locale handling

### PR DESCRIPTION
**Pull request by bdauria**
## Description

Before:
Screenshot from 2020-10-29 14-40-36

After:
Screenshot from 2020-10-29 14-40-47

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)

## Target serie

- [x] 21.04.x (master)

<h2> How this pull request can be tested ? </h2>

- Make sure all dates and times formatting are correctly handled, including those sent to the API.

### CODE REVIEW

File: www/front_src/src/Resources/Actions/api/index.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import formatISO from 'date-fns/formatISO';
 import axios, { AxiosResponse, CancelToken } from 'axios';
-import { map, pick } from 'ramda';
+import { map, pick, startsWith } from 'ramda';
+import dayjs from 'dayjs';
```
Comment (Thebarda) [inline www/front_src/src/Resources/Actions/api/index.ts:3]: dayjs and startsWith are unused here

**Review by Thebarda — COMMENTED** [review]
I changed my timezone to Helsinki (+1 hour from Paris)
When selecting a time for a downtime via the Material time picker dialog, It display the time with the wrong timezone (the browser's timezone here)
https://www.loom.com/share/cf091285d439483f91cf8d0195626f16

**Comment by bdauria** [comment]
> I changed my timezone to Helsinki (+1 hour from Paris)
> When selecting a time for a downtime via the Material time picker dialog, It display the time with the wrong timezone (the browser's timezone here)
> https://www.loom.com/share/cf091285d439483f91cf8d0195626f16

Fixed

**Review by Thebarda — APPROVED** [review]
Tested and LGTM

File: www/front_src/src/Provider/index.tsx [hunk 2]
```diff
@@ -1,18 +1,29 @@
 import * as React from 'react';
 
-import { Provider } from 'react-redux';
-import { pick, pathEq, toPairs, pipe, reduce, mergeAll } from 'ramda';
+import 'dayjs/locale/en';
+import 'dayjs/locale/pt';
+import 'dayjs/locale/fr';
```
Comment (kduret) [inline www/front_src/src/Provider/index.tsx:5]: missing spanish

**Review by kduret — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**